tests/test.py

```diff
@@ -429,6 +428,11 @@ def fork_configui(self, proxy_port: int, http_port: int):
         process = self.fork(lambda: subprocess.Popen(args))
         return process
 
+    def get_connector_tls_params(self):
+        return [
+            '--tls',
+            '--tls_sni=acraserver',
```

Comment:
we should not forget to rename this to `acra-server` in `rename` branch

Reply:
it doesn't matter here. at first - it is tests. and it's depends on values in certificate that used in tests

Reply:
oh right, it's a value inside certificate 🙈